include/wx/generic/treectlg.h

```diff
@@ -360,7 +360,7 @@ class WXDLLIMPEXP_CORE wxGenericTreeCtrl : public wxTreeCtrlBase,
     wxDECLARE_NO_COPY_CLASS(wxGenericTreeCtrl);
 };
 
-#if !defined(__WXMSW__) || defined(__WXUNIVERSAL__)
+#if !defined(__WXMSW__) && ! defined(__WXQT__) || defined(__WXUNIVERSAL__)
```

Comment:
We should test for `wxHAS_GENERIC_TREECTRL` here.

Reply:
The QT build is similar to the MSW build in that it currently includes both the native and the generic versions of the tree control.  

 Shall I remove the generic version from the QT build?